README.rst

```diff
@@ -5,12 +5,35 @@ This is a command-line interface for interacting with connexions content. The to
 
 This software requires:
 
- - Python >= 3.5
- - JRE >= 6
+- Python >= 3.5
+- JRE >= 6
+
+Install
+=======
+
+1. Install `git`
```

Comment:
[Enumerated lists in reStructuredText](http://www.sphinx-doc.org/en/stable/rest.html#lists-and-quote-like-blocks)

You can test the output of the rST locally via `rst2html.py` which is part of [docutils](https://pypi.python.org/pypi/docutils). The commandline utility will inform you of errors in the format.